### PR TITLE
Fixup errors in bindings.

### PR DESCRIPTION
**Pull request by fsoikin**
Turns out that the build script in this repo doesn't actually build anything, unless the version has changed, so I didn't catch these errors in my previous PRs.

### CODE REVIEW

**Comment by alfonsogarciacaro** [comment]
Yes, at the moment the build script only publishes the packages (if the RELEASE_NOTES has changed), I actually noticed the errors when trying to publish them yesterday but didn't have time to fix them. I need to add an option to build everything to just catch the errors. Thanks for the fixes, I'll publish the packages right away! 👍

File: Node/ChildProcess.fs [hunk 1]
```diff
@@ -35,7 +35,7 @@ type ExecOptions = {
     /// Timeout, in milliseconds. Default is zero (no timeout).
     timeout: int option
 }
-    with static member Default = { encoding = None; cwd = None; timeout = None; maxBuffer = None }
+    with static member Default = { encoding = None; cwd = None; timeout = None }
```
Comment (fsoikin) [inline Node/ChildProcess.fs]: Could this be saved with a clever `[<Emit>]`?

```
with 
   [<Emit( "{}" )>]
   static member Default : ExecOptions = jsNative
```

Comment (alfonsogarciacaro) [inline Node/ChildProcess.fs]: Hmm, that won't be different than using createEmpty. I'd rather keep the "only interfaces in the bindings" rules for simplicity for the moment. We can think of a better solution later. 

We can also create a Fable. Helpers.Node package and add this kind of things.

Comment (alfonsogarciacaro) [inline Node/ChildProcess.fs]: Hmm, that won't be different than using createEmpty. I'd rather keep the "only interfaces in the bindings" rules for simplicity for the moment. We can think of a better solution later. 

We can also create a Fable. Helpers.Node package and add this kind of things.

Comment (alfonsogarciacaro) [inline Node/ChildProcess.fs]: Hmm, that won't be different than using createEmpty. I'd rather keep the "only interfaces in the bindings" rules for simplicity for the moment. We can think of a better solution later. 

We can also create a Fable. Helpers.Node package and add this kind of things.

**Pull request merged**